src/init.js

```diff
@@ -18,7 +18,15 @@ const init = () => {
       type: 'input',
       name: 'version',
       message: 'Version of your add-on?',
-      default: '1.1.1'
+      default: '1.1.1',
+      validate: function(value) {
+      var pass = value.match((\d*\.)+\d*);
+      if (pass) {
```

Comment:
Use `const` instead of `var`. Read about `semantic versioning` and its pros. Here is a regex that can be used for semantic versioning validation - https://github.com/semver/semver/issues/232 

I don't think we need to incorporate all of the above regex into our code. We can trim it down.

Reply:
Thanks for suggestion :)